fmn/rules/taskotron.py

```diff
@@ -10,7 +10,8 @@ def taskotron_result_new(config, message):
     """
     return message['topic'].endswith('taskotron.result.new')
 
-@hint(categories=['taskotron'], invertible=False)
+
+@hint(categories=['taskotron'])
```

Comment:
I think all of the ``@hint`` instances here will need ``invertible=False``.  Some explanation:

The hint is a hint to datanommer that allows us to show *examples* of matched messages in the web ui when a user is constructing a new filter.  The hint doesn't do *exactly* what the rule does.. but it is a *hint* to try and help optimize that list of example messages.  So, here, the hint is that -- if this rule is in place, go ahead and only query for ``taskotron`` messages and then we'll pass them all through this rule to further narrow down after that.

Invertibility of a hint is a property that, when True, allows you to use the exact opposite of the hint.  If the *negation* of this rule was in place on a filter, could you rule out **all** taskotron messages?  No, because there exists a subset of taskotron messages which do not match this rule.  So, the hint is not invertible.

Again, I think this is the case for all hints added in this PR.